installer/templates/phx_web/components/layouts/app.html.heex

```diff
@@ -60,7 +60,7 @@
 
 <main class="px-4 py-20 sm:px-6 lg:px-8">
   <div class="mx-auto max-w-2xl">
-    {@inner_content}
+    {render_slot(@inner_block)}
   </div>
 </main>
```

Comment:
Do we want to keep it a file on disk or move into the Layouts module?

Do we want to move `flash_group` as a component to the Layouts module and reuse it from the home page?